src/components/Explore/TokenTable/TokenRow.tsx

```diff
@@ -175,20 +169,23 @@ const NameCell = styled(Cell)`
   padding-left: 8px;
   min-width: 200px;
   gap: 8px;
+
+  @media only screen and (max-width: ${SMALL_MEDIA_BREAKPOINT}) {
+    padding-right: 8px;
+  }
 `
 const PriceCell = styled(DataCell)`
   padding-right: 8px;
 `
 const PercentChangeCell = styled(DataCell)`
```

Comment:
need the padding-right: 8px here no matter what

Reply:
sg done